andela/nazgul-cfh
# #156100318 Front-end Karma Unittests Setup

#### What does this PR do?
This PR allows other contributors to this project to seamlessly write their front end unit tests.

#### Description of Task to be completed?
N/A
#### How should this be manually tested?
* There is a dummy tests in the `frontend.spec.js` file in the unittests directory.
* contributor(s) should run the command `npm run test:karma` in the terminal
* After the test has ran in the terminal, the chrome browser would automatically pop-up 
* when chrome pops-up, click on the gray `debug` button on the top right of the window to see the tests

#### Any background context you want to provide?
* NONE
#### What are the relevant pivotal tracker stories?
#156100318
#### Screenshots (if appropriate)
#### Questions: N/A
